src/cli/Cli.ts

```diff
@@ -151,46 +151,8 @@ export class Cli {
         }
       }
     }
-
-    // validate options
-    // validate required options
-    for (let i = 0; i < this.commandToExecute.options.length; i++) {
-      if (this.commandToExecute.options[i].required &&
-        typeof this.optionsFromArgs.options[this.commandToExecute.options[i].name] === 'undefined') {
-        return this.closeWithError(`Required option ${this.commandToExecute.options[i].name} not specified`, this.optionsFromArgs, true);
-      }
-    }
-
-    const optionsWithoutShorts = Cli.removeShortOptions(this.optionsFromArgs);
-    try {
-      // replace values staring with @ with file contents
-      Cli.loadOptionValuesFromFiles(optionsWithoutShorts);
-    }
-    catch (e) {
-      return this.closeWithError(e, optionsWithoutShorts);
-    }
-
-    try {
-      // process options before passing them on to validation stage
-      await this.commandToExecute.command.processOptions(optionsWithoutShorts.options);
-    }
-    catch (e: any) {
-      return this.closeWithError(e.message, optionsWithoutShorts, false);
-    }
-
-    // if output not specified, set the configured output value (if any)
-    if (optionsWithoutShorts.options.output === undefined) {
-      optionsWithoutShorts.options.output = this.getSettingWithDefaultValue<string | undefined>(settingsNames.output, undefined);
-    }
-
-    const validationResult: boolean | string = this.commandToExecute.command.validate(optionsWithoutShorts);
-    if (typeof validationResult === 'string') {
-      return this.closeWithError(validationResult, optionsWithoutShorts, true);
-    }
-
-    return Cli
-      .executeCommand(this.commandToExecute.command, optionsWithoutShorts)
-      .then(_ => process.exit(0), err => this.closeWithError(err, optionsWithoutShorts));
+    const shouldPrompt = this.getSettingWithDefaultValue<string | undefined>(settingsNames.prompt, undefined)?.toLowerCase() === 'true';
```

Comment:
Default value shouldn't be `undefined`. If we assume that by default the option is disabled, then we should assume `false` when the value has not been configured.

Reply:
ok done